tests/macsec/test_fault_handling.py

```diff
@@ -170,7 +174,10 @@ def check_mka_establishment():
         # The mka should be establishing or established
         # To check whether the MKA establishment happened within 90 seconds
         assert not wait_until(90, 1, 12, check_mka_establishment), (
-            "MKA establishment failed. Expected MKA to not establish within expected time, but it did. "
+            "MKA establishment failed.Expected MKA to not establish within expected time, but it did."
+            "Current status: '{}'.".format(
+                check_mka_establishment()
```

Comment:
it does have failed reason in this assertion, run check_mka_establishment() again is not a good idea I think. Please remove this line